src/app/deposit/deposit-dialog/deposit-dialog.component.ts

```diff
@@ -95,9 +100,25 @@ export class DepositDialogComponent implements OnInit {
   getUniswapLink$(amount: StablecoinBigNumber) {
     return this.swapUniswapService.getLink(amount)
   }
+
+  /**
+   * Some fiat to crypto exchanges like Ramp don't ensure we'll receive
+   * the exact amount of proposed cryptocurrency, so we need to raise the amount
+   * to have a safety margin that will surpass the minimum allowed investment value.
+   * @private
+   */
+  private adjustDepositAmount(amount: StablecoinBigNumber, min?: StablecoinBigNumber): StablecoinBigNumber {
+    if (!min) return amount
+
+    const safetyMargin = this.conversion.toStablecoin('1')
```

Comment:
Is it possible to extract this value to some config or something similar?

Reply:
I rather wouldn't, at least for now. It's a nasty workaround that we're not sure whether it even works. But generally, I agree.